mail/api.py

```diff
@@ -159,22 +161,13 @@ def messages_for_recipients(recipients_and_contexts, template_name):
 
 def send_messages(messages):
     """
-    Sends the messages
+    Sends the messages and logs any exceptions
 
     Args:
         messages (list of EmailMultiAlternatives): list of messages to send
-
-    Raises:
-        SendEmailsException: error if any messages failed to send and why
     """
-    failed_message_errors = []
     for msg in messages:
         try:
             msg.send()
-        except Exception as exc:  # pylint: disable=broad-except
-            failed_message_errors.append(
-                (msg, exc)
-            )
-
-    if failed_message_errors:
-        raise SendEmailsException(failed_message_errors)
```

Comment:
I don't believe we're using this anymore, could you remove the exception definition itself too while you're in here?

Reply:
Already did (`mail/exceptions.py` removed).